modules/installation/pages/uyuni-proxy-setup.adoc

```diff
@@ -0,0 +1,252 @@
+
+
+
+[[proxy.pxe.dhcp]]
+=== Configure DHCP for PXE through Proxy
+
+{productname} uses Cobbler for client provisioning.
+PXE (tftp) is installed and activated by default.
+Clients must be able to find the PXE boot on the {productname} Proxy using DHCP.
+Use this DHCP configuration for the zone which contains the clients to be provisioned:
+
+----
+next-server: <IP_Address_of_Proxy>
+filename: "pxelinux.0"
+----
+
+
+////
+// REMARK: 2019-08-23, ke: this needs closer checking
+[[replacing.a.susemgrproxy]]
+== Replace a {susemgrproxy}
+
+A proxy does not contain any information about the clients that are connected to it.
+Because of this, a proxy can be replaced by a new one at any time.
+The replacement proxy must have the same name and IP address as its predecessor.
+
+Leave the old proxy in place while you prepare the replacement.
+Create a reactivation key for this system and then register the new proxy using the reactivation key.
+If you do not use the reactivation key, you will need to re-register all the clients against the new proxy.
+
+[[proxy.migration3.replace]]
+.Procedure: Replacing a proxy and Keeping the Clients Registered
+. Before starting the actual migration procedure, save the data from the old proxy, if needed.
+Consider copying important data to a central place that can also be accessed by the new proxy.
+// REMARK: 2019-08-23, ke: this is unclear
+// ** Copy the scripts that are still needed.
+// ** Copy the activation keys from the previous server.
+// Of course, it is always better to re-create the keys.
+. Shut down the proxy.
+. Install a new proxy{nbsp}{productnumber}, following xref:install-proxy-unified.adoc[Proxy Installation].
+. In the {productname} {webui} select the newly installed {susemgrproxy} and delete it from the systems list.
+[[step.at.proxy.migration3.replace.react]]
+. In the {webui}, create a reactivation key for the old proxy system: On the System Details tab of the old proxy click [guimenu]``Reactivation``.
+Click [guimenu]``Generate New Key``, and make a note of the new key, as you will need it later.
+For more information about reactivation keys, see xref:reference:systems/system-details/sd-reactivation.adoc[Reactivation Keys].
+. After the installation of the new proxy, you might also need to:
+** Copy the centrally saved data to the new proxy system
+** Install any other needed software
+** Set up TFTP synchronization if the proxy is used for autoinstallation
+
+[IMPORTANT]
+====
+During the installation of the proxy, clients will not be able to reach the {productname} Server.
+After a proxy system has been deleted from the systems list, all clients connected to this proxy will be (incorrectly) listed as `directly connected` to the {productname} Server.
+After the first successful operation on a client, such as execution of a remote command or installation of a package or patch, this information will automatically be corrected.
+This may take some hours.
+====
+////
```

Comment:
Is this comment needed?

Reply:
this is the comment end marker...  asciidoc decoration.